asmcli/lib/validate.sh

```diff
@@ -329,10 +329,14 @@ EOF
 exit_if_apis_not_enabled() {
   local ENABLED; ENABLED="$(get_enabled_apis)";
   local REQUIRED; REQUIRED="$(required_apis)";
+  local OLD_REQUIRED; OLD_REQUIRED="$(old_required_apis)";
   local NOTFOUND; NOTFOUND="";
 
   info "Checking required APIs..."
   NOTFOUND="$(find_missing_strings "${REQUIRED}" "${ENABLED}")"
+  OLD_NOTFOUND="$(find_missing_strings "${OLD_REQUIRED}" "${ENABLED}")"
+
+  if [[ -z "${OLD_NOTFOUND}" ]]; then return; fi
```

Comment:
Can we give a warning like "mesh.googleapis.com API will be required in a future version of ASM"?

Reply:
Good idea, added.